src/braket/pennylane_plugin/ops.py

```diff
@@ -456,3 +456,60 @@ def compute_matrix(phi_0, phi_1):
     def adjoint(self):
         (phi_0, phi_1) = self.parameters
         return MS(phi_0 + np.pi, phi_1, wires=self.wires)
+
+
+class AAMS(Operation):
+    r""" AAMS(phi_0, phi_1, theta, wires)
+
+    IonQ native Arbitrary-Angle Mølmer-Sørenson gate.

```

Comment:
Do we want to refer to this specifically as IonQ's? The MS gate is an ion trap native gate, and although this one is IonQ's parametrization, it still makes sense more broadly.

Reply:
The existing native gates are identified as IonQ's; if you think this is worth updating, I'd prefer to update all of them in a separate PR